wallet/public/index.html

```diff
@@ -60,19 +37,66 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Agoric Wallet</title>
+    <script type="module">
+      const MAX_ATTEMPTS = 10;
+      const WAIT_PER_ATTEMPT = 10;
+      let attempts = 0;
+      const startWhenKeplrAvailable = () => {
+        console.debug('looking for keplr');
+        attempts += 1;
+        if (window.keplr) {
+          // Allow the React dev environment to extend the console for debugging
+          // features.
+          const consoleTaming =
+            '%NODE_ENV%' === 'production' ? 'safe' : 'unsafe';
```

Comment:
This test defaults unsafe, which is too dangerous. I see that this problem is not new to this PR, but it does need to be fixed. All configurations options which have a default behavior should default to the safe behavior. If not fixed in this PR, which would be understandable, please file an issue.

Reply:
That's fine to address here [11e24ef](https://github.com/Agoric/wallet-app/pull/82/commits/11e24ef9bae6b9038760f64199792fd1a57afb6e)